### PR TITLE
Incremented perfdash version

### PR DESCRIPTION
**Pull request by pawbana**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Incremented prefdash version since new changes to it where made here: https://github.com/kubernetes/perf-tests/pull/1992.

### CODE REVIEW

File: perfdash/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 all: push
 
 # See deployment.yaml for the version currently running-- bump this ahead before rebuilding!
-TAG = 2.37
+TAG = 2.38
 
 REPO = gcr.io/k8s-testimages
```
Comment (marseel) [inline perfdash/Makefile:6]: Could you also change repo to k8s-staging-perf-tests, please?

Comment (pawbana) [reply perfdash/Makefile]: Done

**Pull request merged**